JetMETCorrections/Type1MET/interface/CorrectedMETProducerT.h

```diff
@@ -82,6 +84,13 @@ class CorrectedMETProducerT : public edm::stream::EDProducer<>
     delete algorithm_;
   }
     
+  static void fillDescriptions(edm::ConfigurationDescriptions& descriptions) {
+    edm::ParameterSetDescription desc;
+    desc.add<std::string>("@module_label");
+    desc.add<edm::InputTag>("src");
+    descriptions.addDefault(desc);
```

Comment:
I am not familiar with the configurayion description tool, it was made following other JetMET examples, I am going to correct it